lib/ReactViews/Workbench/Controls/ChartItemSelector.jsx

```diff
@@ -7,8 +7,10 @@ import ChartView from "../../../Charts/ChartView";
 import React from "react";
 import Chartable, { axesMatch } from "../../../Models/Chartable";
 
-const ChartItem = observer(({ item, chartItem }) => {
-  const lineColor = chartItem.getColor();
+export const ChartItem = observer(({ item, chartItem }) => {
+  const lineColor = chartItem.isSelectedInWorkbench
+    ? chartItem.getColor()
+    : "#fff";
```

Comment:
This is a text color which should match the color of the chart line but if the line is unselected use a default color instead of "acquiring" a new color from the palette. This frees up the color for use by other active chart items.